### PR TITLE
367 navbar header restyle

### PR DESCRIPTION
**Pull request by katiescruggs**
Travis appears to be failing due to Chromium browser crashing - not sure how to fix.

### CODE REVIEW

File: client/src/components/Footer.js [hunk 1]
```diff
@@ -5,6 +5,9 @@ import { Link } from 'react-router';
 const Footer = () => (
   <Grid fluid className="footer-container">
     <Col xs={12}>
+      <p>Questions? Reach out to us!</p>
```
Comment (hoop71) [inline client/src/components/Footer.js]: This font could be a bit smaller? It's larger than some of the body font above.

Comment (hoop71) [inline client/src/components/Footer.js]: Should there be a link or contact info here?

File: client/src/components/Instructions.js [hunk 2]
```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+
+const Instructions = () => (
```
Comment (hoop71) [inline client/src/components/Instructions.js]: Love your clean, SFCs :)

File: client/src/components/InfoAndLinks.js [hunk 3]
```diff
@@ -3,8 +3,9 @@ import RecyclingInfo from '../components/RecyclingInfo';
 import PropManagerLinks from '../components/PropManagerLinks';
 
 const InfoAndLinks = () => (
-  <div>
+  <div className="info-and-links">
```
Comment (hoop71) [inline client/src/components/InfoAndLinks.js]: ** [class-wrapper-names] ** We have been trying to use a wrapper syntax for these types of classes. Should we consider that here too?  I know the app has lots of inconsistencies. Thoughts?

File: client/src/components/Instructions.js [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+  <div className="Instructions">
```
Comment (hoop71) [inline client/src/components/Instructions.js]: ** [class-wrapper-names] ** We have been trying to use a wrapper syntax for these types of classes. Should we consider that here too?

File: client/src/components/Navbar.js [hunk 5]
```diff
@@ -19,7 +19,7 @@ function UserAuthNav(props) {
   if (props.auth && (!props.auth.googleID && !props.auth.facebookID)) {
     return (
       // if not logged in > show sign in options
-      <NavDropdown id="tools-dropdown" eventKey={5} title="Login">
+      <NavDropdown id="tools-dropdown" eventKey={5} title="LOGIN">
```
Comment (hoop71) [inline client/src/components/Navbar.js]: Should we make the NavBar headings once signed in all caps as well, My Campaign and Sign Out, {user}?

File: client/src/components/RecyclingInfo.js [hunk 6]
```diff
@@ -4,23 +4,18 @@ import { Col, PageHeader } from 'react-bootstrap';
...
 const RecyclingInfo = () => (
   <Col xs={10} xsOffset={1} md={6} mdOffset={0} className="home-section-container">
-    <div>
-      <PageHeader className="home-section-page-header">Wait, But Why?</PageHeader>
-    </div>
-    <div>
-      <h3 className="home-section-title">
-        Denver's recycling rate is only 20%, while the national average is 35%.
-      </h3>
+    <div className="home-section-div">
```
Comment (hoop71) [inline client/src/components/RecyclingInfo.js]: ** [class-wrapper-names] ** We have been trying to use a wrapper syntax for these types of classes. Should we consider that here too?

File: client/src/components/StepByStep.js [hunk 7]
```diff
@@ -18,33 +18,29 @@ Step.propTypes = {
...
-        description="Request recycling services from your landlord by sharing your building's campaign page."
-      />
-      <Step
-        emojiIcon="&#x1F64C;"
-        title="Recycle!"
-        description="Recycle and live in a more sustainable building."
-      />
-    </Row>
-  </Col>
+  <div className="StepByStep">
```
Comment (hoop71) [inline client/src/components/StepByStep.js]: ** [class-wrapper-names] ** We have been trying to use a wrapper syntax for these types of classes. Should we consider that here too?

File: client/src/components/StepByStep.js [hunk 8]
```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import { Row, Col, PageHeader } from 'react-bootstrap';
+import { Col, PageHeader } from 'react-bootstrap';
 
 const Step = props => (
```
Comment (hoop71) [inline client/src/components/StepByStep.js]: Is this function being used anymore?

File: client/src/components/StepByStep.js [hunk 9]
```diff
@@ -18,33 +18,29 @@ Step.propTypes = {
...
-        description="Recycle and live in a more sustainable building."
-      />
-    </Row>
-  </Col>
+  <div className="StepByStep">
+    <PageHeader className="home-section-page-header info-graphic-header">
+      HOW DOES THIS WORK?
+    </PageHeader>
+    <div className="info-graphic">
+      <div className="box">
```
Comment (hoop71) [inline client/src/components/StepByStep.js]: Consider making this a function to dry up the code?

File: client/src/components/Footer.js (hunk 1 above)
Comment (katiescruggs) [reply client/src/components/Footer.js]: Totally agree - changed to 14px.

File: client/src/components/Navbar.js (hunk 5 above)
Comment (katiescruggs) [reply client/src/components/Navbar.js]: Yes, thanks! Changed those.

File: client/src/components/StepByStep.js (hunk 9 above)
Comment (katiescruggs) [reply client/src/components/StepByStep.js]: Sure, which section would you recommend making a function?

File: client/src/components/InfoAndLinks.js (hunk 3 above)
Comment (katiescruggs) [reply client/src/components/InfoAndLinks.js]: Sure, I think it would be great to keep that consistent.

**Pull request merged**